credentialdigger/client.py

```diff
@@ -1378,3 +1381,64 @@ def update_similar_snippets(self,
                 self.update_discovery(d['id'], state)
                 n_updated_snippets += 1
         return n_updated_snippets
+
+    def assign_categories(self, discoveries):
```

Comment:
this is not a relevant for the Client. Why should we add it? It's only related to the UI rendering, that's why it's in `server.py`

Reply:
It has been added here because it is being called by the export_discoveries_csv